rawler/data/lenses.toml

```diff
@@ -158,6 +158,13 @@ aperture_range = [[14, 10], [14, 10]]
 
 # Canon EF-S mount lenses
 
+[[lenses]]
+key = "EF-S24mm f/2.8 STM"
+make = "Canon"
+model = "EF-S24mm f/2.8 STM"
```

Comment:
The correct model name is: "EF-S 24mm f/2.8 STM"

Reply:
fixed